pkg/utils/emails/builders.go

```diff
@@ -54,17 +55,15 @@ const (
 	DateFormat  = "Monday, January 2, 2006"
 )
 
-// EmailData includes data which is required for all emails
-type EmailData struct {
-	SenderName     string
-	SenderEmail    string
-	RecipientName  string
-	RecipientEmail string
+// Contacts includes the sender and recipient of an email
+type Contacts struct {
+	Sender    sendgrid.Contact
+	Recipient sendgrid.Contact
```

Comment:
I'm trying to think about how this data structure might evolve in the future; will we always just have one sender and one recipient or will we want multiple recipients? If so, will multiple recipients be on the same email or will a new email be created for each of them? 

Because of these questions - I think I actually prefer the `EmailData` struct name better; perhaps the `Subject` could also be in `EmailData`? That might be a more evolvable way of dealing with the information needed to send an email in the future.